tests/misc.sh

```diff
@@ -191,6 +191,17 @@ supported()
 			return 1
 		fi
 		;;
+	link)
+		r=`${fstest} create a 0644 2>/dev/null | tail -1`
+		if [ ${r} != 0 ]; then
+			return 0
+		fi
+		r=`${fstest} link a b 2>/dev/null | tail -1`
+		rm -f a b
+		if [ ${r} == "EOPNOTSUPP" ]; then
```

Comment:
You shouldn't try to autodetect support, since verifying support is supposed to be the whole point of pjdfstest.  Instead, select support based on the os and/or filesystem name.

Reply:
Done.